pages/getting-started/testing-your-app.md

```diff
@@ -0,0 +1,490 @@
+```
+
+Assim que o arquivo for criado, você deve vê-lo na lista de arquivos de especificação.
+
+![home page spec](https://docs.cypress.io/img/guides/testing-your-app-home-page-spec.049e17f2.png)
+
+Agora você precisará adicionar o seguinte código em seu arquivo de teste para acessar seu servidor: 
+
+```javascript
+describe('The Home Page', () => {
+  it('successfully loads', () => {
+    cy.visit('http://localhost:8080') // change URL to match your dev URL
+  })
+})
+```
+
+Agora clique no arquivo `home_page_spec.js` e observe o Cypress abrir seu navegador.
+
+Se você se esqueceu de iniciar o servidor, verá o erro abaixo: 
+
+![visit fail](https://docs.cypress.io/img/guides/testing-your-app-visit-fail.10d4b941.png)
+
+Se você iniciou seu servidor, deverá ver sua aplicação carregada e funcionando. 
+
+## Passo 3: Configure Cypress
+
+[//]: <> (TODO - Adicionar link opção de configuração)
+Se você pensar no futuro, perceberá rapidamente que você digitará muito essa URL, 
+já que todo teste precisará visitar alguma página da sua aplicação. 
+Felizmente, o Cypress fornece uma 
+[opção de configuração](https://docs.cypress.io/guides/references/configuration.html) para isso. 
+Vamos aproveitar isso agora.
+
+Abra seu [arquivo de configuração](https://docs.cypress.io/guides/references/configuration.html) 
+(`cypress.json` no seu diretório do projeto, por padrão) ele começa vazio:
+
+```javascript
+{}
+```
+
+Vamos adicionar a opção `baseUrl`.
+
+```javascript
+{
+  "baseUrl": "http://localhost:8080"
+}
+```
+
+[//]: <> (TODO - Adicionar links dos comandos)
+Isso irá prefixar automaticamente os comandos [cy.visit()](https://docs.cypress.io/api/commands/visit.html) 
+e [cy.request()](https://docs.cypress.io/api/commands/request.html) com esta baseUrl. 
+
+> Sempre que você modificar seu arquivo de configuração, o Cypress irá reiniciar-se automaticamente e 
+encerrar todos os navegadores abertos. Isto é normal. Clique no arquivo de especificações novamente para 
+reiniciar o navegador.
+
+Nós agora podemos visitar o caminho relativo, e omitir o nome do host e a porta.
+
+```javascript
+describe('The Home Page', () => {
+  it('successfully loads', () => {
+    cy.visit('/')
+  })
+})
+```
+
+Excelente! Tudo ainda deve estar verde.
+
+> Opções de Configuração
+
+[//]: <> (TODO - Adicionar link configuração)
+O Cypress tem muito mais opções de configuração que você pode usar para personalizar seu comportamento. 
+Coisas como onde seus testes residem, períodos de tempo limite padrão, variáveis de ambiente, qual relatório usar, etc.
+Verifique-os em [Configuração](https://docs.cypress.io/guides/references/configuration.html)!
+
+## Estratégias de Teste
+
+Você está prestes a embarcar na escrita de testes para sua aplicação e somente você conhece sua aplicação, 
+então não temos muitos conselhos específicos para lhe dar.
+
+**O que testar, onde estão os casos extremos e as costuras, quais regressões você provavelmente encontrará, etc.,** 
```

Comment:
@lucastfa-toptal I agree, but what's the best option to translate "where the edge cases and seams are" ?

Reply:
No idea what the author meant with "seams" in this context :D maybe it is better to remove and keep only "casos extremos" because "costuras" doesn't make any sense in our QA context :D

Reply:
I think the author uses 'seams' to refer to how components integrate with each other. What about using 'e as integracões'?

Reply:
Makes sense, I like it! :)

Reply:
done